ccm_web/client/src/pages/AddNonUMUsers.tsx

```diff
@@ -0,0 +1,163 @@
+import React, { useEffect, useState } from 'react'
+import { Backdrop, CircularProgress, Grid, makeStyles, Typography } from '@material-ui/core'
+
+import * as api from '../api'
+import ErrorAlert from '../components/ErrorAlert'
+import Help from '../components/Help'
+import MultipleUserEnrollmentWorkflow from '../components/MultipleUserEnrollmentWorkflow'
+import UserEnrollmentForm from '../components/UserEnrollmentForm'
+import MethodSelect from '../components/MethodSelect'
+import usePromise from '../hooks/usePromise'
+import {
+  CanvasCourseSection, CanvasCourseSectionWithCourseName, getRolesUserCanEnroll, injectCourseName,
+  sortSections
+} from '../models/canvas'
+import { CCMComponentProps } from '../models/FeatureUIData'
+
+const useStyles = makeStyles((theme) => ({
+  root: {
+    padding: 25,
+    textAlign: 'left'
+  },
+  spacing: {
+    marginBottom: theme.spacing(2)
+  },
+  container: {
+    position: 'relative',
+    zIndex: 0
+  },
+  backdrop: {
+    zIndex: theme.zIndex.drawer + 1,
+    color: '#FFF',
+    position: 'absolute',
+    textAlign: 'center'
+  }
+}))
+
+type InputMethod = 'single' | 'csv'
```

Comment:
I see that `single` and `csv` is used multiple place in this file. I feel it should be extracted as a constant and use it

Reply:
Okay, @pushyamig, I think I've kind of resolved this by using a string enum. What do you think? https://github.com/tl-its-umich-edu/canvas-course-manager-next/pull/270/commits/4131f2871bc83e46cd6698b6db47e5cf629f2598

Reply:
Nice